mbof/views.py

```diff
@@ -46,7 +46,7 @@ class MessageViewSet(viewsets.ModelViewSet):
     """
```

Comment:
`class MessageViewSet` (line not shown in the diffs) should be `class EventViewSet`.